Readme.md

```diff
@@ -12,46 +12,43 @@ to automate publishing releases (both full & preview releases) - see
 [**Making a Release**](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/making-a-release.md).
 
 ## Goal
-This library will help you load the configuration of your application from S3 or the SSM parameter store.
+This library will help you find and load the configuration of your application from the SSM parameter store.
```

Comment:
I've slightly deemphasised the S3 functionality in the docs, as I believe SSM is recommended for new apps.  If that's not true, I can change it back.